src/microbe_stage/MicrobeHUD.cs

```diff
@@ -733,7 +735,8 @@ public void UpdateEnvironmentalBars(BiomeConditions biome)
 
         sunlightLabel.GetNode<Label>("Value").Text =
             string.Format(CultureInfo.CurrentCulture, percentageFormat, sunlightPercentage);
-        temperature.GetNode<Label>("Value").Text = averageTemperature + " °C";
+        temperatureLabel.GetNode<Label>("Value").Text =
+            averageTemperature + " " + temperature.Unit;
```

Comment:
Here should use the translation as well. Actually a few lines above this, the percentageFormat use shows well how the unit format should also be made to work.